src/sentry/utils/cloudfunctions.py

```diff
@@ -18,10 +18,45 @@
 WRAPPER_JS = """
 const userFunc = require('./user.js');
 Object.assign(process.env, require('./env.json'));
+const Sentry = require("@sentry/node");
+const Tracing = require("@sentry/tracing");
+exports.start = async (message, context) => {
+  if (process.env.SENTRY_DSN) {
+    Sentry.init({
+      dsn: process.env.SENTRY_DSN,
+      integrations: [
+        new Sentry.Integrations.Http({ tracing: true }),
+      ],
+      tracesSampleRate: 1.0,
+    });
+  }
+  if (!userFunc) {
+    console.error("Your code needs to export a function. module.export = () => {}");
+    return;
+  }
+  if (typeof userFunc !== 'function') {
+    console.error("Your code needs to export a function. Instead, " + typeof userFunc + " was exported.");
```

Comment:
```suggestion
    console.error("Your code needs to export a function. Instead, " + typeof userFunc + " was exported. Make sure your code contains an export like `module.export = () => { /* your code here */ }`");
```

We might want to spell out what is required more clearly.